src/common/services/service-config.ts

```diff
@@ -3,3 +3,14 @@ import axios from 'axios';
 export const axiosInstance = axios.create({
   baseURL: import.meta.env.VITE_API_BASE_URL,
 });
+axiosInstance.defaults.withCredentials = true;
```

Comment:
```suggestion
export const axiosInstance = axios.create({
  baseURL: import.meta.env.VITE_API_BASE_URL,
  withCredentials: true
});
```
사소한 건데 axios 인스턴스 생성 시 옵션으로 withCredentials를 true로 설정가능합니다!
개인적으로는 옵션 내에 withCredentials를 설정하는 게 인지하기가 쉽더라구요!
여러 인스턴스가 있는 경우 어떤 인스턴스의 withCredentials가 true인지 알기도 좀 더 편한 것 같기도 하구요!